### PR TITLE
Week2_PR2

### PR DESCRIPTION
**Pull request by weidajiang**
Update:
1. Created RandomMotto servlet, give random motto back to front
2. Used async request to fetch data in javascript

### CODE REVIEW

**Review by ReshmaMary-gcloud — CHANGES_REQUESTED** [review]
Could you add the javascript changes as well

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -111,8 +112,9 @@ <h3>My Story</h3>
                         is a big problem in any circumstance. Therefore, I took many ESL courses, and then went to Borough of Manhattan Community College
                         in order to make my American dream come true. In the last four years, I have learned a lot and met many people with similar
                         backgrounds. We are all striving hard to achieve our dreams.</p>
-                    <h3>Motto</h3>
-                    <h5>The attitude towards things determines someone’s destiny, and the key to success is the ability to capture details</h5>
+                    <!--> add a entry for calling my js function <-->
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/webapp/index.html:115]: Take care of indentation

**Review by ReshmaMary-gcloud — APPROVED** [review]
Thank you for updating. Looks good

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (weidajiang) [reply portfolio/src/main/webapp/index.html:115]: Thank you so much

**Pull request merged**